DynamicBlurView/CGImage+Accelerate.swift

```diff
@@ -62,4 +63,39 @@ extension CGImage {
 
         return context?.makeImage(with: blendColor, blendMode: blendMode, size: size)
     }
+    
+    func isARG8888() -> Bool {
+        return bitsPerPixel == 32 &&
+            bitsPerComponent == 8 &&
+            (bitmapInfo.rawValue & CGBitmapInfo.alphaInfoMask.rawValue) != 0
+    }
+    
+    func createARGBBitmapContext() -> CGContext? {
+        
+        let width = self.width
+        let height = self.height
+        
+        let bitmapBytesPerRow = width * 4
+        let bitmapByteCount = bitmapBytesPerRow * height
+        
+        let colorSpace = CGColorSpaceCreateDeviceRGB()
+        
+        let bitmapData = malloc(bitmapByteCount)
+        if bitmapData == nil {
+            return nil
+        }
+        
```

Comment:
Allocated memory has to deallocate.
Refs [CGImage+Accelerate.swift#L43-L47](https://github.com/alessandrogarcez/DynamicBlurView/blob/c7358ea0f3d4848462e5a9eeb09048f15eae1339/DynamicBlurView/CGImage%2BAccelerate.swift#L43-L47)